doc/manuals/taskvine/index.md

```diff
@@ -1792,6 +1792,10 @@ change.
 | resource-submit-multiplier | Assume that workers have `resource x resources-submit-multiplier` available.<br> This overcommits resources at the worker, causing tasks to be sent to workers that cannot be immediately executed.<br>The extra tasks wait at the worker until resources become available. | 1 |
 | wait-for-workers        | Do not schedule any tasks until `wait-for-workers` are connected. | 0 |
 | wait-retrieve-many      | Rather than immediately returning when a task is done, `m.wait(timeout)` retrieves and dispatches as many tasks<br> as `timeout` allows. Warning: This may exceed the capacity of the manager to receive results. | 0 |
+| main-loop-max-receives | Sets the max number of receives for the manager to do in the main loop. This is receives of either `receive_one_task` or `receive_all_tasks_from_worker` | 1 |
+| main-loop-prefer-receives | If set to 1, main loop restarts after receiving at least one task | 0 |
```

Comment:
```
prefer-receives | If set to 1, receive all completed tasks from all workers before dispatching new tasks to workers. If set to 0 then...
```
Let's avoid "main-loop" and "restarts", as these terms do not have any meaning unless you are familiar with the code.

Reply:
Also, does `wait-retrieve-many` has any relevance? Should it be deleted?